acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -13,6 +13,7 @@ module InstallUtils
         :solaris_10    => /solaris-10/,
         :solaris_11    => /solaris-11/,
         :windows       => /windows/,
+        :eos           => /^eos-/,
```

Comment:
Thanks for fixing this one up @LuvCurves. Can you apply the same to *all* the regexes in this PR? I have the same concern for the skips and confines.